### DISCUSSION TITLE
Welcome to RomWBW Discussions!

### DISCUSSION THREAD

**Discussion by wwarthen** [wwarthen/RomWBW#162] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fos1** [comment]
Good afternoon,

I am fairly new to the RC2014 / SC126 group. I use RomWBW on an SC126. I
have a number of questions as the group becomes active. I have a RomWBW
module on order for the RC2014 Pro.

Thank you for the invitation.

Jeff

**Reply by wwarthen** [reply]
> Good afternoon, I am fairly new to the RC2014 / SC126 group. I use RomWBW on an SC126. I have a number of questions as the group becomes active. I have a RomWBW module on order for the RC2014 Pro. Thank you for the invitation. Jeff

Hi Jeff,

Did you actually get an email from GitHub?  I just enabled the discussions function for the project.  I didn't actually realize that GitHub would send an invitation to people.  Regardless, I'm happy for your participation, but I hope I didn't spam a bunch of people inadvertently.

-Wayne

**Comment by fos1** [comment]
Hi Wayne,

Yes, I did receive the Github email. It is not a problem for me. I just
began using your software a few weeks ago.

Thank you, Jeff

**Comment by wrljet** [comment]
I received an announcement email from GitHub.  But I am "watching all activity" for the project.

Bill

**Reply by wwarthen** [reply]
> I received an announcement email from GitHub. But I am "watching all activity" for the project.

Ah, maybe that explains it.  I'm just sensitive to sending emails to people that haven't requested them.
